.devops/repo/src/commands/build.rs

```diff
@@ -0,0 +1,32 @@
+use crate::utils::{read_applications, repo_root};
+use std::{env::set_current_dir, process::Command};
+
+pub(crate) fn run() {
+    let root = repo_root();
+    let apps = read_applications();
+
+    let mut errors: Vec<String> = vec![];
+
+    for app in apps {
+        println!("\nBuilding {}", app);
+
+        set_current_dir(format!("{}/{}/project", root, app));
+
+        let build = Command::new("forc").arg("build").status();
```

Comment:
instead of using `set_current_dir` on the parent process, use 
```
let working_dir = std::fs::canonicalize(format!("{}/{}/project", root, app)?;

let build = Command::new("forc")
    .current_dir(working_dir)
    .arg("build")
    .status();
```